Added sitemap for collections & taxonomies (#78)
rapidez/statamic: README.md

After you're done running the install command make sure to check our configuration guide written [here](https://docs.rapidez.io/3.x/packages/statamic.html).

### Sitemap

> It is possible to create a sitemap per store, each sitemap is an index containing all collections and taxonomies that do have views. The XML files will be generated as:

Reviewer: Is this still true?

When you create a form you could use `rapidez-statamic::emails.form` as HTML template which uses the [Laravel mail template](https://laravel.com/docs/master/mail#customizing-the-components) with all fields in a table, make sure you enable markdown!

### Sitemap

> It is possible to create a sitemap per store, each sitemap is a index containing all collections & taxonomies that contain their own views.
> The XML files will be generated as:

Reviewer 2: ```suggestion
It is possible to create a sitemap per store, each sitemap is an index containing all collections and taxonomies that do have a views. The XML files will be generated as:
```

```shell

> sitemap_statamic_{site_handle}
> sitemap_statamic_collection_{site_handle}_{collection_handle}
> sitemap_statamic_taxonomy_{site_handle}_{taxonomy_handle}

Reviewer 2: ```suggestion
/sitemap_statamic_{site_handle}.xml
/sitemap_statamic_collection_{site_handle}_{collection_handle}.xml
/sitemap_statamic_taxonomy_{site_handle}_{taxonomy_handle}.xml
```

```
You can generate the sitemaps by running:
```shell

> rapidez:statamic:generate:sitemap

Reviewer: add `php artisan`
